scripts/devtron-reference-helm-charts/cronjob-chart_1-2-0/app-values.yaml

```diff
@@ -130,14 +132,16 @@ server:
 servicemonitor:
   additionalLabels: {}
 
-imagePullSecrets: []
+imagePullSecrets:
```

Comment:
Why these changes are done at multiple places? Can you please review the extra changes which are not intended.

Reply:
@gunish-dt Any update on this?

Reply:
looking into it